doc/classes/Vector4.xml

```diff
@@ -1,27 +1,34 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="Vector4" version="4.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
 	<brief_description>
+		Vector used for 4D math using floating point coordinates.
 	</brief_description>
 	<description>
+		4-element structure that can be used to represent any quadruplet of numeric values.
+		It uses floating-point coordinates. See [Vector4i] for its integer counterpart.
+		[b]Note:[/b] In a boolean context, a Vector4 will evaluate to [code]false[/code] if it's equal to [code]Vector4(0, 0, 0)[/code]. Otherwise, a Vector4 will always evaluate to [code]true[/code].
```

Comment:
```suggestion
		[b]Note:[/b] In a boolean context, a Vector4 will evaluate to [code]false[/code] if it's equal to [code]Vector4(0, 0, 0, 0)[/code]. Otherwise, a Vector4 will always evaluate to [code]true[/code].
```